source/base/vectorization.cc

```diff
@@ -30,31 +30,31 @@ static_assert(std::is_standard_layout<VectorizedArray<float>>::value &&
                 std::is_trivial<VectorizedArray<float>>::value,
               "VectorizedArray<float> must be a POD type");
 
-#if DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 1 && !defined(DEAL_II_MSVC)
-#  if DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 3 && defined(__AVX512F__)
+#if DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 128 && !defined(DEAL_II_MSVC)
```

Comment:
Do you know why we exclude MSVC? This should have a comment here.

Reply:
Some systems require the explicit instantiation of the `static constexpr` variant, others like MSVC do explicitly not want it. Let me find the respective change in the history.